Building.md

```diff
@@ -140,7 +142,7 @@ and possibly clean it up a bit, curating the information for the target audience
 You can get the latest released version with:
 
 ```bash
-git describe --tags --abbrev=0
+git describe --abbrev=0
```

Comment:
`git describe --tags --abbrev=0` was resolving all tags, but we only need the first one reachable from master.
Now with more pre-releases it was resolving a tag from one of the prereleases as